.github/workflows/reusable_workflow.yaml

```diff
@@ -79,34 +76,60 @@ jobs:
           archetype: ${{ inputs.archetype }}
           req_check_only: ${{ inputs.multi_arch }}
 
-      - name: Set up Docker Buildx for multi-arch
-        if: ${{ inputs.multi_arch }}
-        uses: docker/setup-buildx-action@v3
-
-      - name: Login to Quay for multi-arch
+      - name: Log in to quay.io for multi-arch
         if: ${{ inputs.multi_arch }}
-        uses: docker/login-action@v3
+        uses: redhat-actions/podman-login@v1
         with:
           registry: quay.io
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
-
-      - name: Build and push multi-arch image for release
+      
+      - name: Build multi-arch image for release
+        id: build-multi-arch-release
         if: github.event_name == 'release' && inputs.multi_arch == true
```

Comment:
Yes, but in separate jobs, all of the other prerequisite actions have to be repeated instead of just the `if` statements. With Github actions I find it difficult to find reasonable ways to avoid a lot of repetition when you have conditionals.